README.md

```diff
@@ -8,7 +8,7 @@
 
 _NOTE: The transition from 1.8.0 (or before) to 1.8.1 (or after) requires uninstalling Airflow before installing the new version. The package name was changed from `airflow` to `apache-airflow` as of version 1.8.1._
 
-Airflow is a platform to programmatically author, schedule and monitor
+Airflow is a platform to programmatically author, schedule, and monitor
```

Comment:
Wow. Controversial! This is kind of funny, but if you are serious about this then you'll have to go through all the docs rst and change them all, and even then I have no idea how you plan on enforcing this peculiar grammar idiosyncrasy. If you're not serious then this PR is noise for the committers who I can see have a huge backlog to get through.